app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,21 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %title InstagramChallenge
+    %link{:rel => 'stylesheet', :href => "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css"}
```

Comment:
Line is too long. [113/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace